### PR TITLE
add LB: haship and rand

### PR DESCRIPTION
**Pull request by qiangmzsx**
1.add LB: haship and rand
2.change fasthttp.Request to fasthttp.RequestCtx

### CODE REVIEW

**Comment by qiangmzsx** [comment]
1.add LB: haship and rand
2.change fasthttp.Request to fasthttp.RequestCtx

**Comment by qiangmzsx** [comment]
1.add LB: haship and rand
2.change fasthttp.Request to fasthttp.RequestCtx

**Comment by qiangmzsx** [comment]
1.add LB: haship and rand
2.change fasthttp.Request to fasthttp.RequestCtx

File: pkg/lb/rand.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	if size < 1 {
+		return 0
+	}
+	server := servers[this.Random.Intn(size)]
+	return server.ID
+}
+
+func init() {
+	//Register("hash", newHashIPBalancer())
+}
```
Comment (zhangxu19830126) [inline pkg/lb/rand.go]: removed

**Pull request closed without merging**